components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -1675,6 +1675,8 @@ void ConversationHandler::BuildAssociatedContentInfo() {
     const GURL url = associated_content_delegate_->GetURL();
     metadata_->associated_content->hostname = url.host();
     metadata_->associated_content->url = url;
+    metadata_->associated_content->content_id =
```

Comment:
just fyi this whole thing is a bit of an anti-pattern. Using a builder here would be better.